File: api/budget4home/Models/ExpenseModel.cs
using System;

namespace budget4home.Models
{
    public enum ExpenseType
    {
        Incoming = 0,
        Outcoming
    }

    public class ExpenseModel : BaseModel
    {
        public ExpenseType Type { get; set; }
        public decimal Value { get; set; }
        public DateTime Date { get; set; }
        public string Comments { get; set; }
    }
}

File: src/models/expenseModel.ts
export enum ExpenseType {
  Incoming = 0,
  Outcoming = 1
}
export interface ExpenseModel {
  id: number;
  type: ExpenseType;
  name: string;
  value: number;
  date: Date;
  comments: string;
  scheduleBy: number;
  scheduleTotal: number;

  /** relations */
  labelId: number;
  labelName: string;
}
